.prettierignore

```diff
@@ -7,3 +7,4 @@ test/tapes
 storybook-static
 pnpm-lock.yaml
 nuxt-template-overrides
+#src/components/VSkipToContentContainer.vue . Ignore this file because of conflicting behavior with eslint
```

Comment:
We don't ignore the whole file at all though :thinking: Wouldn't the comment make more sense to only mention alongside the specific `prettier-ignore` pragma rather than referencing the entire file (which is not ignored)?